map_view/display-layer-view-state/src/main/java/com/esri/samples/display_layer_view_state/DisplayLayerViewStateSample.java

```diff
@@ -101,60 +93,50 @@ public void start(Stage stage) {
         EnumSet<LayerViewStatus> layerViewStatus = statusChangeEvent.getLayerViewStatus();
         displayViewStateText(layerViewStatus);
 
-        // if there is an error or warning, format and display the message as an alert
+        // if there is an error or warning, format and display a message as an Alert
         ArcGISRuntimeException error = statusChangeEvent.getError();
         if (error != null) {
           Throwable cause = error.getCause();
-          String message = (cause != null) ? cause.toString() : error.toString();
+          String message = (cause != null) ? "Network issue, unable to load layer" : error.getMessage();
```

Comment:
Do we know certainly that this is would be a network issue? Otherwise we can just say what's wrong (`unable to load layer`)?